unidist/core/backends/mpi/core/communication.py

```diff
@@ -83,19 +83,70 @@ class MPIState:
     ----------
     comm : mpi4py.MPI.Comm
         MPI communicator.
-    rank : int
+
+    Attributes
+    ----------
+    comm : mpi4py.MPI.Comm
+        Global MPI communicator.
+    host_comm : mpi4py.MPI.Comm
+        MPI subcommunicator for the current host.
+    global_rank : int
         Rank of a process.
-    world_sise : int
+    global_size : int
         Number of processes.
+    host : str
+        IP-address of the current host.
+    topology : dict
+        Dictionary, containing worker rank assignments by IP-addresses in
+        the form: `{"node_ip0": [rank_2, rank_3, ...], "node_ip1": [rank_i, ...], ...}`.
```

Comment:
```suggestion
        the form: `{"node_ip0": [rank_2, rank_3, ...], "node_ip1": [rank_i, ...], ...}`.
        Note that ranks start with 2 because 0 is for master and 1 is for monitor.
```